cli/kata-check_ppc64le.go

```diff
@@ -17,6 +19,11 @@ const (
 	archCPUModelField  = genericCPUModelField
 )
 
+var (
+	ppc64CpuCmd     = "ppc64_cpu"
```

Comment:
Is this command guaranteed to be installed (forever) and in `$PATH`?

Reply:
No.

Reply:
So I think maybe it's better to print a warning to indicate "ppc64_cpu isn't installed, can't detect SMT"?

Reply:
Thanks for the inputs @WeiZhang555 @jodh-intel : I have updated my PR.